### ISSUE TITLE
Helm chart doesn't allow all security contexts to be set

### ISSUE THREAD

**Issue by corang** [DefectDojo/django-DefectDojo#11032]
**Is your feature request related to a problem? Please describe**
When attempting to deploy defectdojo in a secure way on kubernetes there is no mechanism that allows me to set the security context to non root for all pods/containers

**Describe the solution you'd like**
Either the helm chart defaults to non root user/group for all pods and containers or exposes them via values to be able to be set by the end user

**Describe alternatives you've considered**
The alternative is maintaining a custom version of the helm chart

**Comment by dsever** [comment]
You are completely right; it is not implemented in the best way. Unless you want to do it, I can create PR very soon.

**Comment by dsever** [comment]
Please can you check https://github.com/DefectDojo/django-DefectDojo/pull/11152 do I need to understand correctly?

**Comment by kiblik** [comment]
Fixed in #13210, will be released as part of `2.52.0`
